pkg/metadata/metadata_test.go

```diff
@@ -10,11 +10,31 @@ import (
 	"testing"
 )
 
+// generateExpected marshals/unmarshals a supplied metadata object.
+// Rather than hand-curating our own expected data, this functionally
+// achieves the same test result but it's cleaner.
+func generateExpected(m *Metadata) map[string]interface{} {
```

Comment:
The advantage of hard coding the test strings in here is that it forces the developer to be a little more intentional about the design of the JSON. With marshalling the JSON programmatically, it won't catch things like misnamed fields or missed fields.

I think I'd rather have the hard coding here rather than this function.